public/app/features/explore/LiveTailButton.tsx

```diff
@@ -5,14 +5,13 @@ import memoizeOne from 'memoize-one';
 import tinycolor from 'tinycolor2';
 import { CSSTransition } from 'react-transition-group';
 
-import { GrafanaTheme, GrafanaThemeType, useTheme } from '@grafana/ui';
+import { GrafanaTheme, useTheme } from '@grafana/ui';
 
 const getStyles = memoizeOne((theme: GrafanaTheme) => {
-  const orange = theme.type === GrafanaThemeType.Dark ? '#FF780A' : '#ED5700';
-  const orangeLighter = tinycolor(orange)
+  const orangeLighter = tinycolor(theme.colors.orangeDark)
```

Comment:
Isn't this one of the colours Jess defined in new oranges (ref https://github.com/grafana/grafana/pull/19274#discussion_r327526964)? Same with `pulseTextColor` below. If yes, then I suggest introducing these colours to the theme.

Reply:
No it is not, when I console-logged the colours, they were:
#ff943d for orangeLighter
#918378 for pulseTextColor